docs/Makefile

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2017-2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+# Minimal makefile for Sphinx documentation
+
+
+SHELL = /bin/bash
+MAKEDIR := $(dir $(lastword $(MAKEFILE_LIST)))
+
+# You can set these variables from the command line.
+SPHINXOPTS      =
+SPHINXBUILD     = [ -e env/bin/activate ] && source env/bin/activate; sphinx-build
+SPHINXAUTOBUILD = [ -e env/bin/activate ] && source env/bin/activate; sphinx-autobuild
+SPHINXPROJ      = SymbiFlowV2X
```

Comment:
this again looks like something from another repo

Reply:
Fixed.